aws/resource_aws_default_route_table_test.go

```diff
@@ -455,39 +498,82 @@ resource "aws_default_route_table" "test" {
 
 const testAccDefaultRouteTable_vpc_endpoint = `
 resource "aws_vpc" "test" {
-    cidr_block = "10.0.0.0/16"
+  cidr_block = "10.0.0.0/16"
 
   tags = {
-        Name = "terraform-testacc-default-route-table-vpc-endpoint"
-    }
+    Name = "terraform-testacc-default-route-table-vpc-endpoint"
+  }
 }
 
 resource "aws_internet_gateway" "igw" {
-    vpc_id = "${aws_vpc.test.id}"
+  vpc_id = "${aws_vpc.test.id}"
 
   tags = {
-        Name = "test"
-    }
+    Name = "terraform-testacc-default-route-table-vpc-endpoint"
+  }
 }
 
 resource "aws_vpc_endpoint" "s3" {
-    vpc_id = "${aws_vpc.test.id}"
-    service_name = "com.amazonaws.us-west-2.s3"
-    route_table_ids = [
-        "${aws_vpc.test.default_route_table_id}"
-    ]
+    vpc_id          = "${aws_vpc.test.id}"
+    service_name    = "com.amazonaws.us-west-2.s3"
```

Comment:
Can you make this test region independent using the [`aws_region`](https://www.terraform.io/docs/providers/aws/d/region.html) data source?

Reply:
good catch, i was thinking about it and forgot when commiting

Reply:
done